src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
Consider using a constant for the daily cost value instead of a literal to make the code more maintainable and readable, as per the checklist item 2.